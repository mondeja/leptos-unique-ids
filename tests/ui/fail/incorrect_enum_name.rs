use leptos_unique_ids::leptos_unique_ids;

#[leptos_unique_ids]
pub enum Foo {} // This should be called `Ids` instead

fn main() {}
