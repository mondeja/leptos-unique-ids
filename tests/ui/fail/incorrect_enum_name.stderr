error: Expected an enum formed with the token tree `enum Ids {{}}`.
 --> ui/fail/incorrect_enum_name.rs:3:1
  |
3 | #[leptos_unique_ids]
  | ^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `leptos_unique_ids` (in Nightly builds, run with -Z macro-backtrace for more info)
